Description: merge lp:~sdeziel/apparmor/dnsmasq-lxd-bridge:
 .
   Simon Deziel 2016-04-08 usr.sbin.dnsmasq: add lxd-bridge rules
 .
 Acked-by: Christian Boltz <apparmor@cboltz.de>
 .
 References: https://bugs.launchpad.net/ubuntu/+source/apparmor/+bug/1566944
Origin: commit, revision id: apparmor@cboltz.de-20160408222413-rjouiz0iw0w8qv3e
Author: Simon Deziel <simon.deziel@gmail.com>
Author: Christian Boltz <apparmor@cboltz.de>
Last-Update: 2016-04-08
X-Bzr-Revision-Id: apparmor@cboltz.de-20160408222413-rjouiz0iw0w8qv3e

=== modified file 'profiles/apparmor.d/usr.sbin.dnsmasq'
--- old/profiles/apparmor.d/usr.sbin.dnsmasq	2015-12-30 21:23:42 +0000
+++ new/profiles/apparmor.d/usr.sbin.dnsmasq	2016-04-08 20:23:47 +0000
@@ -74,6 +74,10 @@
   /{,var/}run/lxc/dnsmasq.pid    rw,
   /var/lib/misc/dnsmasq.*.leases rw,
 
+  # lxd-bridge pid and lease files
+  /{,var/}run/lxd-bridge/dnsmasq.pid   rw,
+  /var/lib/lxd-bridge/dnsmasq.*.leases rw,
+
   # NetworkManager integration
   /{,var/}run/nm-dns-dnsmasq.conf r,
   /{,var/}run/sendsigs.omit.d/*dnsmasq.pid w,

